guides/v2.3/config-guide/cli/config-cli-subcommands-config-mgmt-set.md

```diff
@@ -267,5 +267,6 @@ Result:
 
 <pre class="no-copy">web/unsecure/base_url - http://example-for-store.com/</pre>
 
-#### Related topic
+{:.ref-header}
+Related topic
```

Comment:
Should be a new line after the header text, right?

Reply:
Ah, yes. I thought I caught all of those. Thanks.